app/views/layouts/application.html.slim

```diff
@@ -29,10 +29,11 @@ html
       }
 
     #main
-      - if content_for?(:nav)
-        = yield(:nav)
-      - else
-        = render("nav")
+      - if !request.original_url.include? "/partners/"
```

Comment:
I think instead of this we should simply check to see if the user is a `partner?`

Reply:
I would prefer to do it that way, but we need to provide legacy nav on the dashboard, for partners as well as publishers.